project/Dependencies.scala

```diff
@@ -14,30 +14,37 @@ object Dependencies {
   lazy val scalaStmVersion = settingKey[String]("The version of ScalaSTM to use.")
   lazy val scalaCheckVersion = settingKey[String]("The version of ScalaCheck to use.")
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
+  lazy val sslConfigVersion = settingKey[String]("The version of ssl-config to use.")
   val junitVersion = "4.12"
-  val sslConfigVersion = "0.3.7"
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
   val aeronVersion = "1.15.1"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.12.8", "2.13.0-M5"),
+    crossScalaVersions := Seq("2.12.8", "2.13.0-RC1"),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
-    scalaStmVersion := sys.props.get("akka.build.scalaStmVersion").getOrElse("0.9"),
+    scalaStmVersion := sys.props.get("akka.build.scalaStmVersion").getOrElse("0.9.1"),
     scalaCheckVersion := sys.props
         .get("akka.build.scalaCheckVersion")
         .getOrElse(CrossVersion.partialVersion(scalaVersion.value) match {
           case Some((2, n)) if n >= 12 => "1.14.0" // does not work for 2.11
           case _                       => "1.13.2"
         }),
-    scalaTestVersion := "3.0.7",
+    scalaTestVersion := "3.0.8-RC2",
     java8CompatVersion := {
       CrossVersion.partialVersion(scalaVersion.value) match {
+        // java8-compat is only used in a couple of places for 2.13,
+        // it is probably possible to remove the dependency if needed.
         case Some((2, n)) if n >= 13 => "0.9.0"
         case Some((2, n)) if n == 12 => "0.8.0"
         case _                       => "0.7.0"
       }
-    })
+    },
+    sslConfigVersion := {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n >= 13 => "0.4.0"
```

Comment:
Why is that needed? Is 0.4.0 missing for other Scala version?

Reply:
I think this was just a 'conservative choice' because I wasn't sure 0.4.0 is binary compatible with 0.3.7, so I wasn't sure if updating to 0.4.0 for 2.11/2.12 would be an acceptable change.